contributing/workflow/bisecting_regressions.rst

```diff
@@ -81,57 +71,27 @@ Determine the commit hashes
 
 To start bisecting, you must first determine the commit hashes (identifiers) of
 the "bad" and "good" build. "bad" refers to the build that exhibits the bug,
-whereas "good" refers to the version that doesn't exhibit the bug. If you're
-using a pre-release build as the "good" or "bad" build, browse the `download
-mirror <https://downloads.tuxfamily.org/godotengine/>`__, go to the folder that
-contains the pre-release you downloaded and look for the ``README.txt`` file.
-The commit hash is written inside that file.
-
-If you're using a stable release as the "good" or "bad" build, use one of the
-following commit hashes depending on the version:
-
-.. code-block:: none
-
-    4.1.1-stable
-    4.1-stable
-    4.0.3-stable
-    4.0.2-stable
-    4.0.1-stable
-    4.0-stable
-    3.5.2-stable
-    3.5.1-stable
-    3.5-stable
-    3.4.5-stable
-    3.4.4-stable
-    3.4.3-stable
-    3.4.2-stable
-    3.4.1-stable
-    3.4-stable
-    3.3.4-stable
-    3.3.3-stable
-    3.3.2-stable
-    3.3.1-stable
-    3.3-stable
-    3.2-stable
-    3.1-stable
-    3.0-stable
-
-You can also use this Bash function to retrieve the Git commit hash of a
-pre-release build (add it to your ``$HOME/.bashrc`` or similar):
-
-::
-
-    gd_snapshot_commit() {
-        curl -s https://downloads.tuxfamily.org/godotengine/$1/$2/README.txt \
-            | grep 'from commit' \
-            | sed 's/^Built from commit \(.*\)\.$/\1/'
-    }
-
-Example usage:
+whereas "good" refers to the version that doesn't exhibit the bug. 
 
-.. code-block:: shell
+You can use either a commit hash (like ``06acfccf8``), the tag of a stable
+release (like ``4.2.1-stable``), or a branch like ``master``.
 
-    gd_snapshot_commit 4.0 beta4
+If you're using a pre-release build as the "good" or "bad" build, you can find
+the commit hash in the Project Manager in the lower-right corner, or in in the
+**Help > About Godot** dialog in the Godot editor. The version information will
+look something like ``v4.4.beta3.official [06acfccf8]``, and the commit hash is
+within the brackets, in this case ``06acfccf8``. You can click on the version
+information to copy it, including the commit hash.
+
+Alternately, you can browse the `Godot Archive
+<https://godotengine.org/download/archive/>`__, and find the commit hash within
+the release page linked from the **News** button.
+
+If you're using a stable release as the "good" or "bad" build, you can use the
+tag of that release directly, such as ``4.2-stable`` or ``4.2.1-stable``. A full
+list of release tags is listed `on GitHub
```

Comment:
```suggestion
list of release tags is available `on GitHub
```

Feels weird to say that a list is listed somewhere. Other than that this change looks good!